docs/en/setup/envoy/als_setting.md

```diff
@@ -87,3 +88,14 @@ this analyzer requires Istio to enable the metadata exchange plugin (you can ena
 or if you're using Istio 1.7+ and installing it with profile `demo`/`preview`, it should be enabled then).
 
 The [blog](https://skywalking.apache.org/blog/obs-service-mesh-vm-with-sw-and-als/) illustrates the detail of how it works, and a step-by-step tutorial to apply it into the [Online Boutique](https://github.com/GoogleCloudPlatform/microservices-demo) system.
+
+### `persistence`
+
+`persistence` analyzer adapts the Envoy access log format to
+SkyWalking's [native log format](https://github.com/apache/skywalking-data-collect-protocol/blob/master/logging/Logging.proto)
+, and forwards the formatted logs to [LAL](../../concepts-and-designs/lal.md), where you can configure persistent
+conditions, such as `sampler`, only persist error logs, etc.
+
+**Attention**: because `persistence` analyzer also needs a mechanism to map the logs into responding services, hence,
+you need to configure at least one of `k8s-mesh` or `mx-mesh` as its antecedent so that `persistence` analyzer knows which service the
+logs belong to.
```

Comment:
Do you mean, we need to set `SW_ENVOY_METRIC_ALS_HTTP_ANALYSIS=k8s-mesh,persistence`?

Reply:
> Do you mean, we need to set `SW_ENVOY_METRIC_ALS_HTTP_ANALYSIS=k8s-mesh,persistence`?

Yes, something like `k8s-mesh,persistence`, `mx-mesh,persistence`, or `k8s-mesh,mx-mesh,persistence`

Reply:
Let's add this as an example in the doc. I assume this, because I know how this config works, but for the end users, I think they don't know what it means by `at least one of `k8s-mesh` or `mx-mesh` as its antecedent`.